packages/js/src/ui/Inbox.tsx

```diff
@@ -31,7 +31,21 @@ const InternalInbox = () => {
   return (
     <div class={style('novu', 'root')}>
       <div class="nt-text-2xl nt-font-bold">{t('inbox.title')}</div>
-      <BellContainer />
+      <div class="nt-absolute nt-top-1/2 nt-left-1/2 nt-transform -nt-translate-x-1/2 -nt-translate-y-1/2">
+        <div>
```

Comment:
Is this extra div required? If not, let's remove it. We need to keep HTML as clean as possible and to the absolute minimum. Every extra HTML node, makes customization and the appearance prop DX handler.

Reply:
Additionally, we should be adding a descriptor to every element, unless it something we specifically don't want customized (like a Novu banner).